query-builder/src/main/java/com/datastax/oss/driver/api/querybuilder/schema/RelationOptions.java

```diff
@@ -58,6 +58,20 @@ default SelfT withCDC(boolean enabled) {
     return withOption("cdc", enabled);
   }
 
+  /**
+   * Defines the crc check chance.
+   *
+   * <p>Note that using this option with a version of Apache Cassandra less than 5.0 will raise a
```

Comment:
It looks like this was added in 3.0: https://github.com/apache/cassandra/blob/115c5539840354d6af81a246d391071d2851f636/NEWS.txt#L1472-L1473 as part of CASSANDRA-9839

```suggestion
   * <p>Note that using this option with a version of Apache Cassandra less than 3.0 will raise a
```